controllers/tutorial.controller.js

```
@@ -0,0 +1,74 @@
+/* eslint-disable no-console */
+const { response } = require('express');
+const { Tutorial } = require('../models/tutorial.schema');
+const mangoose = require('../init/db');
+
+// create Document or insert
+const createTutorial = async (req, res) => {
+  try {
+    const demoTutorial = new Tutorial({
+      id: "1",
+      published: true,
+    });
+    const result = await demoTutorial.save();
+    // eslint-disable-next-line no-console
+    console.log(result);
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.log(err);
+  }
+};
+
+const getTutorialById = async (req, res) => {
+  const findId = req.params.id;
+  try {
+    const tutorials = await Tutorial.find({ id: findId });
+    res.json(tutorials);
+  } catch (error) {
+    res.json({ message: error });
+  }
+};
+
+const getTutorial = async (req, res) => {
+  try {
+    const tutorials = await Tutorial.find();
+    res.json(tutorials);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log("error");
```

We should return a proper error response from here.